shared/templates/service_disabled/tests/service_disabled.pass.sh

```diff
@@ -2,11 +2,15 @@
 # packages = {{{ PACKAGENAME }}}
 
 SYSTEMCTL_EXEC='/usr/bin/systemctl'
-"$SYSTEMCTL_EXEC" stop '{{{ DAEMONNAME }}}.service'
-"$SYSTEMCTL_EXEC" disable '{{{ DAEMONNAME }}}.service'
-"$SYSTEMCTL_EXEC" mask '{{{ DAEMONNAME }}}.service'
+# Some services use <name>@.service style that is not meant to be activated at all,
+# and only used via socket activation.
+if "$SYSTEMCTL_EXEC" list-unit-files '{{{ DAEMONNAME }}}.service' | grep -q '^{{{ DAEMONNAME }}}.service'; then
+    "$SYSTEMCTL_EXEC" stop '{{{ DAEMONNAME }}}.service'
+    "$SYSTEMCTL_EXEC" disable '{{{ DAEMONNAME }}}.service'
+    "$SYSTEMCTL_EXEC" mask '{{{ DAEMONNAME }}}.service'
+fi
 # Disable socket activation if we have a unit file for it
-if "$SYSTEMCTL_EXEC" list-unit-files | grep -q '^{{{ DAEMONNAME }}}.socket'; then
+if "$SYSTEMCTL_EXEC" list-unit-files '{{{ DAEMONNAME }}}.socket' | grep -q '^{{{ DAEMONNAME }}}.socket'; then
```

Comment:
Since the unit is now informed in the for the `sysctl list-unit-files`, the return code will already be `0` if the unit exists and `1` otherwise. So, this `| grep` is no longer necessary. Also valid for the previous condition and the other file.

Reply:
@maage , do you have plans to work on this soon? Once these conditionals are simplified we should be ready to merge this PR.

Reply:
@maage ping! : )